### DISCUSSION TITLE
Replacement of catalog-v0-API

### DISCUSSION THREAD

**Discussion by fegbers** [amzn/selling-partner-api-models#3333] [General]
Hello all,

to replace the old v0-API, we need the following options in the newer API:

- ProductGroup (always in EN regardless of the marketplace, maybe via an extra call for this marketplace)
- PegiRating
- AudienceRating

The necessary part of the answer is the following: [attributesetlisttype](https://developer-docs.amazon.com/sp-api/docs/catalog-items-api-v0-reference#attributesetlisttype)

Does anyone have an answer? 

Thanks and greetings ✌🏼

**Comment by chapmanjw** [comment]
Hi @fegbers,

The product group is now included in the `summaries` data as `websiteDisplayGroup` in Catalog Items v2022-04-01: https://developer-docs.amazon.com/sp-api/docs/catalog-items-api-v2022-04-01-reference.

Audience rating is a bit trickier. In MWS/v0, the `AudienceRating` only worked for a few marketplaces and was mapped to the country-specify attribute. In the US marketplace, for example, it was mapped to `mpaa_rating`, which is available in the `attributes` section of Catalog Items v2022-04-01. In the DE, it was mapped to `fsk_rating`, which is also available in the `attributes` section. For product type-specific attributes like these, the `attributes` section provides the "raw" reconciled attributes that work hand-in-hand with the Product Type Definitions API. We would not be able to provide a static model that works for every product type (which is the short-coming of the MWS/v0 version that only provides a small set of attributes). 

The `pegi_rating` attribute _should_ work the same way here, but I am not seeing evidence that it is. I will do some investigating on my end to see why `pegi_rating` is missing from the rules that define which "raw" attributes are included in the Catalog Item API responses.

Thanks!

**Reply by fegbers** [reply]
Hi @chapmanjw,
thanks for your fast answer!

Is there any reason, why the summaries property doesn't provide the "human redebale way" (as in `displayGroupRanks` in `salesRanks`) besides `websiteDisplayGroup`?
![image](https://user-images.githubusercontent.com/22808985/169642689-78a88532-c2b8-4d94-b091-c8f628f2b408.png)
I could use `title` of the `salesRanks` property if i had the values every time. Unfortunately it is not always the case..


I could find an example with the `fsk_rating` and other country specific associations. Thanks for the explaination. In case if somebody else is struggling, the following is an example:
ASIN: B0088UPIMY
DE:
![image](https://user-images.githubusercontent.com/22808985/169643587-696fa327-4a1b-4507-867d-372c77c6c6b9.png)
UK:
![image](https://user-images.githubusercontent.com/22808985/169643579-b9447a02-3396-4a51-8341-16b51462872c.png)

B01MG4DCMD on the german market place has a pegi rating:
![image](https://user-images.githubusercontent.com/22808985/169643703-90d02e27-aa68-4736-88b9-be7d06d20baf.png)
As you already mentioned, the property is missing. I could only find the USK rating:
![image](https://user-images.githubusercontent.com/22808985/169643889-55610114-ba8b-4ce8-bade-84e28f366dc9.png)

Thanks for investigating!

**Reply by fegbers** [reply]
@chapmanjw Sorry for interruption, any news?

**Reply by chapmanjw** [reply]
@fegbers We recently added (just this week) the display name for the `websiteDisplayGroup` in the summaries:

```
                    "websiteDisplayGroup": "speakers_display_on_website",
                    "websiteDisplayGroupName": "Speakers"
```

For the data in the `attributes` section, those are the "raw" attribute values that you would submit if you were editing an item via the Listings APIs. The display labels for those values are usually available in the Product Type Definitions API (if the product type has been migrated to the API). 

We have heard some feedback on developers looking only for the "display" version of attributes rather than the "raw" versions. I will make a note of this Github discussion on our tracking item for that.

**Reply by fegbers** [reply]
@chapmanjw Perfect, thanks! 

Is it a bug or did I misunderstand the query parameter `locale`? For the A1PA6795UKMFR9 marketplace ID (Germany) and without the query parameter `locale` I get the german name.

![image](https://user-images.githubusercontent.com/22808985/175294548-a264df81-1ee1-4d3d-ae2e-37d3c2f5bf1e.png)

When i set the locale to `en_US`, nothing changes. According the API documentation it should change

> Locale for retrieving localized summaries. Defaults to the primary locale of the marketplace.

Use case: Regardless of the marketplace, we would like to process the same display group to provide an independent search for the user.

**Reply by chapmanjw** [reply]
@fegbers At this time, that value is only available to us in the locale of the marketplace.
